sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 #### Bugs Fixed
 
 #### Other Changes
+* Updated `azure-cosmos` to version `4.41.0`.
```

Comment:
I am questioning whether we should release encryption or just wait for the major version release. @aayush3011  thoughts?

Reply:
We should have one without the breaking change
And then a new major version against the same azure-cosmos version also including the possibly breaking change